tox.ini

```diff
@@ -1,15 +1,67 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
+    lint
     py27
+    py35
```

Comment:
Reminder: there is no Plone version that supports Python 3.5. But this package can be used outside of Plone of course.

Reply:
`meta/config` uses the same Python versions for all packages. As long a we still support Python 2, we cannot use Python 3.6+ features so it is easy to support Python 3.5, too.